### PR TITLE
docs: keep versions information in one place

### PR DESCRIPTION
**Pull request by perk-sumo**

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Deployment Guide
+# Deployment Guide for release v2.4
```
Comment (pmalek-sumo) [inline deploy/README.md:1]: Not sure if that's a good idea, this will get old very quick.

**Review by astencel-sumo — APPROVED** [review]

File: deploy/README.md (hunk 1 above)
Comment (perk-sumo) [reply deploy/README.md:1]: Yes, that's the question what to put here in the main branch.
Maybe it should be `for unreleased version`?

This particular change here was to show how I think we should keep things in release branches.

Comment (perk-sumo) [reply deploy/README.md:1]: Yes, that's the question what to put here in the main branch.
Maybe it should be `for unreleased version`?

This particular change here was to show how I think we should keep things in release branches.

Comment (perk-sumo) [reply deploy/README.md]: Changed.

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**